public/index.html

```diff
@@ -63,14 +66,33 @@ <h1 class="heading-2">Tell us how we can help...</h1>
                   <li>Do you have dependents or require financial assistance?</li>
                 </ul>
                 <h1 class="heading-2">Here's how to contact me...</h1>
-                <label class="field-label" for="phone-2">Cell Phone</label>
-                <input class="text-field w-input" data-name="Phone 2" id="phone-2" name="Phone" maxlength="256" placeholder="e.g. 713 555 1111" required="required" type="text">
-                <label class="field-label" for="email-2">Email (optional)</label>
-                <input class="text-field w-input" data-name="Email 2" id="email-2" name="Email" maxlength="256" placeholder="e.g. you@email.com" type="email">
-                <label class="field-label" for="zip-2">Current Zip Code:</label>
-                <input class="text-field w-input" data-name="Zip 2" id="zip-2" name="Zip" maxlength="256" placeholder="e.g. 77004" required="required" type="text">
-                <label class="field-label" for="housing-2">Housing Situation:</label>
-                <input class="text-field w-input" data-name="Housing 2" id="housing-2" name="Housing" maxlength="256" placeholder="e.g. at George R Brown Shelter -OR- at Home w/ flood damage" type="text">
+                <div class="small-fields">
+                  <div class="field-group">
+                    <label class="field-label" for="name-2">Name <span>(optional)</span></label>
+                    <input class="text-field w-input" data-name="Name 2" id="name-2" name="Name" maxlength="256" type="text">
+                  </div>
+                  <div class="field-group">
+                    <label class="field-label" for="email-2">Email <span>(optional)</span></label>
+                    <input class="text-field w-input" data-name="Email 2" id="email-2" name="Email" maxlength="256" type="email">
+                  </div>
+                  <div class="field-group">
+                    <label class="field-label" for="phone-2">Cell Phone</label>
```

Comment:
One note here, we should maybe just call this Phone Number, in case they want to provide a number that's not a mobile phone.

Reply:
fine w/ me, do you want to make the change? @laddhoffman